src/app-bundles/location-tree-bundle.js

```diff
@@ -91,9 +91,13 @@ function listToTree( sourceList ) {
       list[ locationIdToListIndex[ currentNode.parent_id ] ].nodes.push( currentNode );
 
       // Add a tree path prop to the original list for easy lookup for locations to tree position
-      sourceList[ i ].tree_path = buildTreePath( currentNode );
+      const treePath = buildTreePath( currentNode );
+      currentNode.path = treePath.split( "/" ); // Add path array to node to easily look up whether it is a child of a specific ID.
```

Comment:
what does assigning `currentNode.path` do? I don't see it being used anywhere else and the districts and basins filters work even if we remove this

Reply:
I think the ability to look up the parents of a node will be needed if/when we try to get the tree to also filter itself when the user is filtering by district/basin. Basically, to look up and see if a node is a child of a given district ID or basin ID in the tree.